src/library_webgl.js

```diff
@@ -868,6 +868,34 @@ var LibraryGL = {
       GL.contexts[contextHandle] = null;
     },
 
+    acquireInstancedArraysExtension: function(ctx) {
+      // Extension available from Firefox 26 and Google Chrome 30
+      var ext = ctx.getExtension('ANGLE_instanced_arrays');
+      if (ext) {
+        ctx['vertexAttribDivisor'] = function(index, divisor) { ext['vertexAttribDivisorANGLE'](index, divisor); };
+        ctx['drawArraysInstanced'] = function(mode, first, count, primcount) { ext['drawArraysInstancedANGLE'](mode, first, count, primcount); };
+        ctx['drawElementsInstanced'] = function(mode, count, type, indices, primcount) { ext['drawElementsInstancedANGLE'](mode, count, type, indices, primcount); };
+      }
+    },
+
+    acquireVertexArrayObjectExtension: function(ctx) {
+      // Extension available from Firefox 25 and WebKit
```

Comment:
nit: is there supposed to be a WebKit version here?

Reply:
Sorry, I don't understand?

Do you mean there's supposed to be a WebKit-prefixed version of the extension? Or do you mean that the comment oddly refers to a WebKit variant? Here I transferred the original comments as-is from the old code without changing them. Updated the comment.